apidoc/Titanium/Media/VideoPlayer.yml

```diff
@@ -586,7 +540,42 @@ properties:
         conditions (if the movie is being streamed from a remote source).
     type: Boolean
     default: true
+    
+  - name: backgroundView
+    summary: Sets the background view for customization which is always displayed behind movie content.
+    description: |
+        To create a background behind the movie content on iOS, you can specify a 
+        background view. On iOS, you can set the `backgroundColor` property directly on
+        the video player, but not a background image or background gradient. For this kind
+        of effect, specify a background view. 
+           
+        Note that the background view must be sized explicitly: percentage sizes do not work properly.
+
+        The following code sample creates a video player with a background image:
+
+            var videoPlayer = Ti.Media.createVideoPlayer({
+                height : 300,
+                width : 300,
+                backgroundView : Ti.UI.createView({
+                    backgroundImage: 'videoPlayerBG.png
+                    width : 300,
+                    height : 300
+                }),
+                borderRadius : 20,
+                borderWidth : 2,
+                borderColor : 'blue',
+            });
 
+        Note that if [scalingMode](Titanium.Media.VideoPlayer.scalingMode) is set to 
+        [VIDEO_SCALING_MODE_FILL](Titanium.Media.VIDEO_SCALING_MODE_FILL) or 
+        [VIDEO_SCALING_ASPECT_FILL](Titanium.Media.VIDEO_SCALING_ASPECT_FILL),
+        the playing video will fill the entire video player area, obscuring any background
+        view.
+    platforms: [iphone, ipad]
+    parameters:
```

Comment:
This is generating an error when you do ```node validate.js``` in ```apidoc```
```
[ERROR] ./Titanium/Media/VideoPlayer.yml
Titanium.Media.VideoPlayer: found 1 error(s)!
	properties
		backgroundView
			Invalid key: parameters
```
Most likely because it doesn't expect properties to contain parameters.